pkg/kyverno/apply/command.go

```diff
@@ -342,6 +336,43 @@ func checkMutateLogPath(mutateLogPath string) (mutateLogPathIsDir bool, err erro
 	return mutateLogPathIsDir, err
 }
 
+// getPolicyAccordingToResourcePath - get policies according to the resource path
+func getPolicyAccordingToResourcePath(policyPaths []string) (policies []*v1.ClusterPolicy, err error) {
```

Comment:
this name is confusing as `Resource` is typically used to refer to the Kubertenes resources. 

Can we change to `getPoliciesFromPaths`?